shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -1127,6 +1151,7 @@ \subsection{Chain Transition}
 we extract the block header body \var{bhb}. From \var{bhb} we extract the block
 nonce $\eta$ and the block slot \var{s}. The transition rule itself has no
 preconditions, instead it calls all its subrules.
+Note that \mathsf{UPIEC} is defined in \cite{byron_ledger_spec}.
```

Comment:
I think this non-math-mode `\mathsf{..}` is currently breaking the hydra build

Reply:
ah, thanks!